File: Vorbereitung/Übungen/1/1-6.cs
namespace Uebung;

class Program {
    static void Main(string[] Args) {
        int x = int.Parse(Console.ReadLine());
        int y = int.Parse(Console.ReadLine());

        bool z = GreaterThan(x, y);
        Console.WriteLine(z);
    }
    static bool GreaterThan(int x, int y) {
        return x > y;
    }
}

File: Vorbereitung/Übungen/1/1-6.c
int greaterThan(int a, int b) {
    if (a > b) {
        return 1;
    }
    return 0;
}

int main() {
    int a;
    int b;
    scanf("%d", &a);
    scanf("%d", &b);

    int c = greaterThan(a, b);
    printf("%d", c);
}